sdk/Lusid.Sdk.Tests/Utilities/InstrumentExamples.cs

```diff
@@ -155,6 +155,93 @@ internal static LusidInstrument CreateExampleSwap()
             );
         }
 
+        internal static InterestRateSwaption CreateExampleInterestRateSwaption()
+        {
+                        // CREATE an Interest Rate Swap (IRS)
+            var startDate = new DateTimeOffset(2020, 2, 7, 0, 0, 0, TimeSpan.Zero);
+            var maturityDate = new DateTimeOffset(2030, 2, 7, 0, 0, 0, TimeSpan.Zero);
+
+            // CREATE the flow conventions, index convention for swap
+            var flowConventions = new FlowConventions(
+                scope: null,
+                code: null,
+                currency: "GBP",
+                paymentFrequency: "6M",
+                rollConvention: "MF",
+                dayCountConvention: "Act365",
+                paymentCalendars: new List<string>(),
+                resetCalendars: new List<string>(),
+                settleDays: 2,
+                resetDays: 2
+                );
+
+            var idxConvention = new IndexConvention(
```

Comment:
add some comments pointing the user to where to get the valid values from

Reply:
I'm not sure what you mean. 
The variables inside the index convention is quite flexible. I am not currently aware we do any crazy validation apart from the obvious ones e.g. currency, tenor, day count convention. These fields have their available choices written in their respective classes and/or in error messages.